app/views/runs/_quick_stats.slim

```diff
@@ -0,0 +1,53 @@
+- previous_upload = @run.user.try(:previous_upload_for, @run.category)
+
+article data-turbolinks-temporary=true
+  .row
+    .col-sm-3
+      .statcard.p-3
+        h2.statcard-number
+          span> = format_ms(run.duration_ms(timing))
+          - if previous_upload.present?
+            - if run.duration_ms(timing) <= previous_upload.duration_ms(timing)
+              small.delta-indicator.delta-negative.text-success.tip-top title='Since last PB'
+                = format_ms_casual(previous_upload.duration_ms(timing) - run.duration_ms(timing))
+            - else
+              small.delta-indicator.delta-positive.text-danger.tip-top title='Since last PB'
+                = format_ms_casual(run.duration_ms(timing) - previous_upload.duration_ms(timing))
+        span.statcard-desc Time
+    .col-sm-3
+      .statcard.p-3
+        h3.statcard-number
+          span> = format_ms(run.sum_of_best_ms(timing))
+          - if previous_upload.present?
+            small.delta-indicator.delta-negative.text-success.tip-top title='Since last PB'
+              = format_ms_casual(previous_upload.sum_of_best_ms(timing) - run.sum_of_best_ms(timing))
+        span.statcard-desc = 'Sum of best'
+    .col-sm-3
+      .statcard.p-3
+        h3.statcard-number = format_ms(run.duration_ms(timing) - run.sum_of_best_ms(timing))
+        span.statcard-desc Potential timesave
+    .col-sm-3
+      .statcard.p-3
+        h3.statcard-number = Run.program(run.timer)
+        span.statcard-desc = link_to('Timer', Run.program(run.timer).website, target: '_blank', class: 'text-muted')
+  .row
+    .col-sm-3
+      .statcard.p-3
+        h3.statcard-number
+          span> = run.attempts
+          - if previous_upload.present?
+            small.delta-indicator.delta-positive.tip-top title='Since last PB'
+              = run.attempts - previous_upload.attempts
+        span.statcard-desc = 'Attempt'.pluralize(run.attempts)
+    .col-sm-3
+      .statcard.p-3
+        h3.statcard-number
+          span = format_ms(run.total_playtime_ms(timing))
+          - if previous_upload.present?
+            small.delta-indicator.delta-positive.text-success.tip-top title='Since last PB'
+              = format_ms_casual(run.total_playtime_ms(timing) - previous_upload.total_playtime_ms(timing))
```

Comment:
I think the formatting is a bit messed up for this.

![chrome_2018-06-06_19-22-11](https://user-images.githubusercontent.com/7030913/41074865-f67ce24c-69be-11e8-90a0-183d6eff314d.png)